lib/pengine/unpack.c

```diff
@@ -54,8 +54,8 @@ static void unpack_rsc_op(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
                           pe_working_set_t *data_set);
 static void determine_remote_online_status(pe_working_set_t *data_set,
                                            pe_node_t *this_node);
-static void add_node_attrs(xmlNode *attrs, pe_node_t *node, bool overwrite,
-                           pe_working_set_t *data_set);
+static void add_node_attrs(const xmlNode *attrs, pe_node_t *node,
```

Comment:
`attrs` in declaration, `xml_obj` in definition. Doesn't matter, I'd only fix if changing something else